apps/storybook/.storybook/main.ts

```diff
@@ -24,5 +26,8 @@ const config: StorybookConfig = {
   docs: {
     autodocs: 'tag',
   },
+  env: {
+    NEXT_PUBLIC_LANG: `en`,
+  },
```

Comment:
i think we can remove this chunk, since it seems like the process.env stuff above was the fix